ui/tests/integration/job-page/parts/summary-test.js

```diff
@@ -152,3 +152,130 @@ test('the children diagram lists all children status figures', function(assert)
     });
   });
 });
+
+test('the summary block can be collapsed', function(assert) {
+  this.server.create('job', {
+    createAllocations: false,
+  });
+
+  this.store.findAll('job');
+
+  return wait()
+    .then(() => {
+      this.set('job', this.store.peekAll('job').get('firstObject'));
+
+      this.render(hbs`
+      {{job-page/parts/summary job=job}}
+    `);
+
+      return wait();
+    })
+    .then(() => {
+      click('[data-test-accordion-toggle]');
+      return wait();
+    })
+    .then(() => {
+      assert.notOk(find('[data-test-accordion-body]'), 'No accordion body');
+      assert.notOk(find('[data-test-legend]'), 'No legend');
+    });
+});
+
+test('when collapsed, the summary block includes an inline version of the chart', function(assert) {
+  this.server.create('job', {
+    createAllocations: false,
+  });
+
+  this.store.findAll('job');
+
+  return wait()
+    .then(() => {
+      this.set('job', this.store.peekAll('job').get('firstObject'));
+
+      this.render(hbs`
+      {{job-page/parts/summary job=job}}
+    `);
+
+      return wait();
+    })
+    .then(() => {
+      click('[data-test-accordion-toggle]');
+      return wait();
+    })
+    .then(() => {
+      assert.ok(find('[data-test-allocation-status-bar]'), 'Allocation bar still existed');
+      assert.ok(
+        find('.inline-chart [data-test-allocation-status-bar]'),
+        'Allocation bar is rendered in an inline-chart container'
+      );
+    });
+});
+
+test('the collapsed/expanded state is persisted to localStorage', function(assert) {
+  this.server.create('job', {
+    createAllocations: false,
+  });
+
+  this.store.findAll('job');
+
+  return wait()
+    .then(() => {
+      this.set('job', this.store.peekAll('job').get('firstObject'));
+
+      this.render(hbs`
+      {{job-page/parts/summary job=job}}
+    `);
+
+      return wait();
+    })
+    .then(() => {
+      assert.notOk(window.localStorage.nomadExpandJobSummary, 'No value in localStorage yet');
```

Comment:
I think this is going to result in flaky tests due to localStorage state retention. We initially worked around this in TFE by wiping localStorage in our `moduleForAcceptance` helper, but eventually switched to [ember-window-mock](https://github.com/kaliber5/ember-window-mock).

Reply:
Just my 2 cents here, I'd use mocking in unit or integration tests, but if its not complicated to use the real thing in an acceptance test then I would. All browsers have localStorage so I'd say ideally it'd be used during acceptance - just my take.